src/_buttons.scss

```diff
@@ -225,3 +225,78 @@
     }
   }
 }
+
+
+.button.is-icon-only-button {
+  background-color: transparent;
+  color: lighten($button-icon-only-default-color, 10.5%);
+  padding: 0.5em 0.25em;
+
+  line-height: 1;
+
+  font-size: 1em;
+  width: 2em;
+  height: 2em;
+  text-align: center;
+
+  &.is-small {
+    font-size: 0.9em;
+  }
+  &.is-medium {
+    font-size: 1.1em;
+  }
+  &.is-large {
+    font-size: 1.2em;
+  }
+  &.is-very-large {
+    font-size: 1.4em;
+  }
+  &.is-extremely-large {
+    font-size: 1.6em;
+  }
+
+  &:hover {
+    background-color: lighten($button-primary-color, 30%);
+    color: darken($button-primary-color, 5%);
+  }
+
+  &:active,
+  &.is-active {
```

Comment:
Curious - why both? Feels like there's a reason for having both, but it's not coming to mind immediately.

Reply:
One is active state (button pressed) and the other can be used to mark buttons ss active programmatically. See the button list example for how it looks. This could be used to mark how you voted if you voted.